### PR TITLE
[AUTOMATIC] Release/4.3.1

### PR DESCRIPTION
**Pull request by RCGitBot**
**This is an automatic release.**

### Other Changes
* Upgrade iOS to 4.13.1 and Android to 5.6.1 (#246) via RevenueCat Git Bot (@RCGitBot)
* Update maven-publish-plugin (#245) via Cesar de la Vega (@vegaro)
* Adds parameters to run bump and dependency updates manually instead of approval jobs (#243) via Cesar de la Vega (@vegaro)
* `IntegrationTests`: don't initialize `Purchases` until `SKTestSession` has been re-created (#244) via NachoSoto (@NachoSoto)

### CODE REVIEW

**Review by vegaro — APPROVED** [review]

**Comment by vegaro** [comment]
I had to remove running `androidSourcesJar` and `androidJavadocJar` because they are not included with the publishing plugin anymore. In any case, it looks like they were not necessary as the sources and javadocs look correct when running just `publish`

**Comment by vegaro** [comment]
The iOS deployment gave a strange CDN error in https://circleci.com/gh/RevenueCat/purchases-hybrid-common/3523?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link and I reran it. 

I think it worked though, because I see it now in Cocoapods correctly published. In any case, I retagged to re run the deployment

**Comment by vegaro** [comment]
It failed again, as expected because it was already published. But I think the first run failed due to some Cocoapods issue when returning a success after deploying. 4.3.1 is live in Cocoapods

File: fastlane/Fastfile [hunk 1]
```diff
@@ -330,7 +330,7 @@ platform :android do
 
     gradle(
       tasks: [
-        "androidSourcesJar", "androidJavadocJar", "publish --no-daemon --no-parallel"
+        "publish --no-daemon --no-parallel"
```
Comment (tonidero) [inline fastlane/Fastfile:333]: Well, I guess publish should indeed build the jar file, and I guess it also adds the javadocs to the jar normally so as long as it's tested, I think this is ok.

**Review by tonidero — APPROVED** [review]
LGTM!

**Pull request merged**